### PR TITLE
Components

### PR DESCRIPTION
**Pull request by fantomkis**

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <Routes>
+          <Route path='/' element={<Home />} />
+          <Route path='/friends' element={<Friends />} />
+          <Route path='/family' element={<Family />} />
+          <Route path='/closefr' element={<CloseFriends />} />
+            <Route path='/justpeple' element={<JustPeople />} />
+          
+        </Routes>
+      </main>
+
```
Comment (AlexKovalchuk) [inline src/components/App.tsx:22]: here should be footer

File: src/components/App.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import Home from '../pages/Home';
+import Friends from './Friends/Friends';
+import CloseFriends from './CloseFriends/CloseFriends';
+import JustPeople from './JustPeople/JustPeople';
+
+function App() {
+  return (
+    <>
+    
+      <Navigation/>
```
Comment (AlexKovalchuk) [inline src/components/App.tsx]: You should create component Header and inside it implement navigation

File: src/components/Navigation/Navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import s from "./Navigation.module.css";
+
+const Navigation = () => {
```
Comment (AlexKovalchuk) [inline src/components/Navigation/Navigation.tsx]: Rename it to Header component

**Review by AlexKovalchuk — CHANGES_REQUESTED** [review]

File: src/components/App.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import s from './App.module.css';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <div className={s.wrap} >
+      <Sidebar/>
+      
+      <main>
```
Comment (AlexKovalchuk) [inline src/components/App.tsx]: where is Main.jsx component?
make routes content of the Main

**Review by AlexKovalchuk — CHANGES_REQUESTED** [review]

File: src/components/App.tsx (hunk 4 above)
Comment (fantomkis) [reply src/components/App.tsx]: Done

File: src/components/Main/Main.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+import Sidebar from '../Sidebar/Sidebar';
+function Main() {
+    return (
+        <main>
+            <Sidebar></Sidebar>
+        </main>
```
Comment (AlexKovalchuk) [inline src/components/Main/Main.tsx:23]: here should be Routing functional also

**Review by AlexKovalchuk — CHANGES_REQUESTED** [review]

**Review by AlexKovalchuk — APPROVED** [review]

**Pull request merged**